diff --git a/xbmc/filesystem/DllLibNfs.h b/xbmc/filesystem/DllLibNfs.h
index 9ba8ebc..4b5ba29 100644
--- a/xbmc/filesystem/DllLibNfs.h
+++ b/xbmc/filesystem/DllLibNfs.h
@@ -85,7 +85,6 @@ public:
   virtual int nfs_pread(struct nfs_context *nfs,     struct nfsfh *nfsfh,  uint64_t offset, uint64_t count, char *buf)=0;
   virtual int nfs_pwrite(struct nfs_context *nfs,    struct nfsfh *nfsfh,  uint64_t offset, uint64_t count, char *buf)=0;
   virtual int nfs_lseek(struct nfs_context *nfs,     struct nfsfh *nfsfh,  uint64_t offset, int whence,   uint64_t *current_offset)=0;
-  virtual void nfs_set_streaming_mode(struct nfsfh *nfsfh, uint32_t v)=0;
 };
 
 class DllLibNfs : public DllDynamic, DllLibNfsInterface
@@ -132,7 +131,6 @@ class DllLibNfs : public DllDynamic, DllLibNfsInterface
   DEFINE_METHOD5(int, nfs_pread,     (struct nfs_context *p1, struct nfsfh *p2,  uint64_t p3,   uint64_t p4,  char *p5))
   DEFINE_METHOD5(int, nfs_pwrite,    (struct nfs_context *p1, struct nfsfh *p2,  uint64_t p3,   uint64_t p4,  char *p5))
   DEFINE_METHOD5(int, nfs_lseek,     (struct nfs_context *p1, struct nfsfh *p2,  uint64_t p3,   int p4,     uint64_t *p5))
-  DEFINE_METHOD2(void,nfs_set_streaming_mode,     (struct nfsfh *p1, uint32_t p2))
 
 
 
@@ -178,7 +176,6 @@ class DllLibNfs : public DllDynamic, DllLibNfsInterface
     RESOLVE_METHOD_RENAME(nfs_symlink,   nfs_symlink)
     RESOLVE_METHOD_RENAME(nfs_rename,    nfs_rename)
     RESOLVE_METHOD_RENAME(nfs_link,      nfs_link)      
-    RESOLVE_METHOD_RENAME(nfs_set_streaming_mode,      nfs_set_streaming_mode)
   END_METHOD_RESOLVE()
 };
 
diff --git a/xbmc/filesystem/NFSFile.cpp b/xbmc/filesystem/NFSFile.cpp
index 37358db..610ace8 100644
--- a/xbmc/filesystem/NFSFile.cpp
+++ b/xbmc/filesystem/NFSFile.cpp
@@ -563,8 +563,6 @@ bool CNFSFile::Open(const CURL& url)
   CLog::Log(LOGDEBUG,"CNFSFile::Open - opened %s",url.GetFileName().c_str());
   m_url=url;
   
-  //gNfsConnection.GetImpl()->nfs_set_streaming_mode(m_pFileHandle, 5 * 1024 * 1024);
-
   struct __stat64 tmpBuffer;
 
   if( Stat(&tmpBuffer) )
